lib/system/mmdisp.nim

```diff
@@ -45,8 +45,12 @@ else:
 
 proc raiseOutOfMem() {.noinline.} =
   if outOfMemHook != nil: outOfMemHook()
-  cstderr.rawWrite("out of memory\n")
-  quit(1)
+  when hostOS == "standalone":
+    panic("out of memory\n")
+  else:
+    when not defined(nimNoLibc):
+      cstderr.rawWrite("out of memory\n")
```

Comment:
```suggestion
  elif not defined(nimNoLibc):
    cstderr.rawWrite("out of memory\n")
```

Reply:
This suggestion is not equivalent to what is proposed in this PR. What we want to achieve is to always call `quit` (when not in standalone mode), but only call `cstderr.rawWrite("out of memory\n")` when libc is available.

Reply:
Ah, I missed that.